pkg/statistics/handle/history/history_stats.go

```diff
@@ -67,31 +72,41 @@ func (sh *statsHistoryImpl) RecordHistoricalStatsToStorage(dbName string, tableI
 	return version, err
 }
 
-// RecordHistoricalStatsMeta records stats meta of the specified version to stats_meta_history table.
-func (sh *statsHistoryImpl) RecordHistoricalStatsMeta(tableID int64, version uint64, source string, enforce bool) {
+// RecordHistoricalStatsMeta records the historical stats meta in mysql.stats_meta_history with a single transaction.
+func (sh *statsHistoryImpl) RecordHistoricalStatsMeta(version uint64, source string, enforce bool, tableIDs ...int64) {
 	if version == 0 {
 		return
 	}
-	if !enforce {
-		tbl, ok := sh.statsHandle.Get(tableID)
-		if !ok {
-			return
-		}
-		if !tbl.IsInitialized() {
-			return
+
+	var filteredTableIDs []int64
```

Comment:
filteredTableIDs： The name means more like a collection of tables that have been excepted.

Reply:
Updated it to targetedTableIDs.